My-Portfolio/css/style.css

```diff
@@ -281,6 +274,120 @@ ul {
     background-repeat: no-repeat;
     background-position: center;
   }
+
+  .project-preview {
```

Comment:
- Kindly make sure your Popup stick to the Figma design as per the project requirements, by:

- [ ] on the desktop kindly make sure to respect the margins, as you can see below the details popup doesn't touch the edges of the screen, there is a dark blur overlay behind.

- Figma:
![Screenshot from 2023-01-31 22-26-26](https://user-images.githubusercontent.com/97834160/215877396-d158b771-1a58-49d5-9cd4-fdba745c072f.png)
- Your design:
![Screenshot from 2023-01-31 22-26-12](https://user-images.githubusercontent.com/97834160/215877432-968280a5-eb4a-4ae0-89ae-8456627725f2.png)


- [ ] on mobile,  Kindly put make sure kindly make sure we can scroll the whole screen(popup +overlay) as the details popup must display as a whole.
- Figma:
![Screenshot from 2023-01-31 22-26-46](https://user-images.githubusercontent.com/97834160/215877901-715e1ec2-0301-4739-ad7e-d90fb3845fb0.png)
- Your design:

[Screencast from 01-31-2023 10:27:10 PM.webm](https://user-images.githubusercontent.com/97834160/215878333-1e6e1e42-3451-4d2a-a3f0-617d7c45c39a.webm)